web_client/.gitignore

```diff
@@ -26,3 +26,4 @@ src/CogSettings.*
 src/data_pb.*
 src/data.*
 src/data_pb_service.*
+src/CogTypes.d.ts
```

Comment:
Better to add a empty line at the end of the file (not having it means your editor is not loading the .editorconfig) which is not ideal

Reply:
Got it